R/Design-methods.R

```diff
@@ -3442,7 +3442,7 @@ setMethod("simulate",
           fit = fitDLEList,
           fit_eff = fitEffList,
           sigma2_est = sigma2Estimates,
-          sigma2betaWest = sigma2betaWEstimates,
+          sigma2_beta_west = sigma2betaWEstimates,
```

Comment:
```suggestion
          sigma2_beta_w_est = sigma2betaWEstimates,
```
and similar elsewhere

Reply:
that was my missing suggestion, sorry about that!